winui/Avatar-view/content-types.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
```

Comment:
Description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.